counter.py

```diff
@@ -0,0 +1,19 @@
+import time
+import keyboard
+
+count = int(input("Enter any integer : "))
+#inputint = int(inputint)
+
+def counter (count):
+
+    i = 0
+    while i < count:
+        
+        time.sleep(1)
+        i = i + 1
+        print(i)
+        
+        if keyboard.is_pressed(' '):
```

Comment:
The program only stops if I hold a key not when I tap it.

Reply:
i couldn't able to add this functionality. please suggest!